templates/taxi/car_list.html

```diff
@@ -1,12 +1,16 @@
 {% extends "base.html" %}
-
+{% load crispy_forms_filters %}
 {% block content %}
   <h1>
     Car list
     <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
       Create
     </a>
   </h1>
+  <form method="get" action="" class="form-inline">
+    {{ search_forms|crispy }}
```

Comment:
The variable `search_forms` should likely be `search_form` to match the context variable passed from the view. Ensure that the variable name is consistent with what is provided in the view's context.